lib/utils.js

```diff
@@ -205,6 +205,27 @@ utils.getResourceUID = function getResourceUID(namespace, resource) {
         .update(namespace + resource, 'binary').digest('hex');
 };
 
+/**
+ * Validate request headers to determine if readable characters
+ * @param {object} headers - headers attached to the http request (lowercased)
+ * @return {boolean} - returns true if headers are readable, false if not
+ */
+utils.checkReadableHeaders = function checkReadableHeaders(headers) {
```

Comment:
Yes, I missed that the end of the filter function was after the ```!startsWith```, sorry for annoying ^^